### DISCUSSION TITLE
Change of IP's

### DISCUSSION THREAD

**Discussion by kdmiller45** [mastodon/mastodon#26553] [Q&A]
I recently moved from the United States to Germany and need to change my Sites IP information, I have changed the Server IP, but where all within Mastodon do I need to change them, I currently get the NGINX page. when answering please include name of file and where is it found

Thanks
Keith

**Comment by StrangeMinds1960** [comment]
What have you done so far?  'I have changed the Server IP', where have you done this?  As far as I'm aware, all you need to do is change the IP's of your hostname(s) of your Mastodon instance in  DNS, be advised that this can take up to 48 hours to propagate

Posting the hostname(s) of your Mastodon instance might help us find the problem

**Comment by ryanmio** [comment]
Check your NGINX configuration is updated to reflect the new IP. NGINX config usually located at /etc/nginx/sites-available/your_config
